server/controllers/v1/comment.ts

```diff
@@ -11,7 +11,7 @@ export type CommentsUser = Comment & {
 };
 
 // The maximum number of results to return.
-const MAX_RESULTS = 50;
+const MAX_RESULTS = 10;
```

Comment:
Only 10 posts per query?

Reply:
Yea, I think this is better than having lots of scrolling on the post page